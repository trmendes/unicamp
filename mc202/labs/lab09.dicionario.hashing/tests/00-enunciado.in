i itdoesntmatter
i thewaythatyoutakeit
i itdoesntmatter
i thewaythatyoumakeit
b itdoesntmatteR
b itdoesntmatter
b thewaythatyoumakeit
r thewaythatyoumakeit
b thewaythatyoumakeit
f

