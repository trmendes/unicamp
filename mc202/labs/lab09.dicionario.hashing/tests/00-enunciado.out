nao encontrado
encontrado 0
encontrado 2
nao encontrado
